package.json

```diff
@@ -25,7 +25,9 @@
   },
   "homepage": "https://github.com/parse-community/parse-server-s3-adapter#readme",
   "dependencies": {
-    "aws-sdk": "2.1399.0"
+    "@aws-sdk/client-s3": "^3.623.0",
```

Comment:
Could you install the dependencies again with `npm i -E ...` and for dev dependencies `npm i -ED ...`